### DISCUSSION TITLE
postman "Bad CPU type in executable"

### DISCUSSION THREAD

**Discussion by majent4** [Homebrew/discussions#3892] [Casks] [answered]
```
$ brew info postman
==> postman: 10.1.2 (auto_updates)
https://www.postman.com/
/usr/local/Caskroom/postman/9.31.0 (121B)
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/postman.rb
==> Name
Postman
==> Description
Collaboration platform for API development
==> Artifacts
Postman.app (App)
==> Analytics
install: 11,943 (30 days), 37,927 (90 days), 150,188 (365 days)

$ postman
-bash: /usr/local/bin/postman: Bad CPU type in executable

$ file /usr/local/bin/postman
/usr/local/bin/postman: Mach-O 64-bit executable arm64

$ uname -p
i386
```

Is there a workaround to install the good version of postman ?

**Comment by p-linnane** [comment, marked as answer]
~Homebrew does not support 32-bit CPU's any longer.~

Apologies, I misread your `uname` output.

`postman` is a cask which install a GUI application into the `/Applications` folder. If you're looking for that CLI binary you want `postman-cli`. If you run `brew reinstall --cask postman-cli` it should reinstall with an Intel build.

**Reply by majent4** [reply]
Thank you for your time !

Device uses 64-bit Intel architecture (MacBook Air). I can't figure out why Homebrew installed this version of postman

**Reply by Bo98** [reply]
Looks like a bug on our side: https://github.com/Homebrew/homebrew-cask/pull/135412

**Reply by p-linnane** [reply]
@majent4 The fix @Bo98 made has been merged. Can you try the reinstall command I posted above?

**Reply by majent4** [reply]
It works, thank you !
